draft-birkholz-rats-epoch-markers.md

```diff
@@ -117,6 +130,8 @@ $payload /= strictly-monotonically-increasing-counter
 
 native-rfc3161-TST-info = bytes ;  DER-encoded value of TSTInfo
 
+TST-info-based-on-CBOR-time-tag = "PLEASE DEFINE"
```

Comment:
Dome :-)

```suggestion
TST-info-based-on-CBOR-time-tag = TSTInfo
```

Reply:
Or is it? Seems there is no cbor time-tag in there. @thomas-fossati, wasn't that the initial intent?